doc/classes/TextureLayered.xml

```diff
@@ -61,9 +84,17 @@
 			<argument index="1" name="layer" type="int">
 			</argument>
 			<description>
+				Replaces the existing [Image] data at the given [code]layer[/code] with this new image.
+				The given [Image] must have the same width, height, image format and mipmapping setting (a [code]bool[/code] value) as the rest of the referenced images.
+				If the image format is unsupported, it will be decompressed and converted to a similar and supported [enum Image.Format].
+				The update is immediate: synced with the draw.
 			</description>
 		</method>
 	</methods>
+	<members>
+		<member name="_images" type="Array" setter="create_from_images" getter="_get_images">
```

Comment:
Was this added by doctool, or did you add it manually? ``_images`` is tagged as ``PROPERTY_USAGE_INTERNAL`` and so shouldn't be appearing in docs.

https://github.com/godotengine/godot/blob/80582ffa662bb4dea223003856320ede37365603/scene/resources/texture.cpp#L2009

Reply:
I added member `_images` manually.  
I'll remove it